plugins/AudioFileProcessor/AudioFileProcessorWaveView.cpp

```diff
@@ -480,7 +480,10 @@ void AudioFileProcessorWaveView::reverse()
 
 void AudioFileProcessorWaveView::updateCursor(QMouseEvent * me)
 {
-	const auto pos = position(me);
+	QPoint pos;
```

Comment:
What if `me` is nullptr and pos is undefined, if pos is used in that case, won't this lead to another issue?

Reply:
yes, but pos isn't used in this function in that case. Unfortunately we can't define pos with sane values when there's no mouse event.